### DISCUSSION TITLE
Emacs eglot not showing docs in mini buffer after updating clangd from 16.0.2 to 18.1.3

### DISCUSSION THREAD

**Discussion by wjrforcyber** [clangd/clangd#2048] [General]
Emacs version 29.1
Platform: Centos7 (Yes, clangd is running in patch mode with proper version of glibc)
Can jump to right definition and give highlight in code but there are no more brief information in mini buffer in eldoc mode.

**Comment by HighCommander4** [comment]
Can you share a code example and [clangd logs](https://clangd.llvm.org/troubleshooting.html#gathering-logs)?

**Reply by wjrforcyber** [reply]
Everything is fine now, it seems like there's no space left on the VM we worked on, so the hint in the mini buffer is not available after the manual update of the clangd(although it showed eglot was connected and clangd was managing C++ mode), thanks for the reply. 
Anyone else having this issue please check the usage of the VM.
